core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -299,7 +299,7 @@ object ConfigCommand extends Logging {
       val configsToBeAdded = opts.options.valueOf(opts.addConfig)
         .split("," + pattern)
         .map(_.split("""\s*=\s*""" + pattern, -1))
-      require(configsToBeAdded.forall(config => config.length == 2), "Invalid entity config: all configs to be added must be in the format \"key=val\".")
+      require(configsToBeAdded.forall(config => config.length == 2), "Invalid entity config: all configs to be added must be in the format of \"key=val\". If val is in the format symbol 'val1,val2', you can use \"key=[val1,val2]\".")
```

Comment:
I suggest the following: `"Invalid entity config: all configs to be added must be in the format \"key=val\" or  \"key=[val1,val2]\" to group values which contain commas."`

Reply:
Yes, I have reproduced the commit code